core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -3255,17 +3304,26 @@ private boolean o3Commit(long lag) {
             o3TimestampMax = getTimestampIndexValue(sortedTimestampsAddr, srcOooMax - 1);
 
 
-            // we are going to use this soon to avoid double-copying lag data
+            // we are going to use this soon to avoid double-copying o3MaxLag data
```

Comment:
i would leave this as "lag data"

Reply:
fixed in https://github.com/questdb/questdb/pull/2759/commits/cd55db87ed385ca10a67463e0e120dcb4c10e123